doc/neps/nep-0031-uarray.rst

```diff
@@ -302,7 +302,7 @@ This is different from monkeypatching in a few different ways:
   so there is at least the loose sense of an API contract. Monkeypatching
   does not provide this ability.
 * There is the ability of locally switching the backend.
-* It has been `suggested <http://numpy-discussion.10968.n7.nabble.com/NEP-31-Context-local-and-global-overrides-of-the-NumPy-API-tp47452p47472.html>`_
+* It has been `suggested <https://mail.python.org/archives/list/numpy-discussion@python.org/thread/X3IJH4M4YUYRF6N5XZKL7XT6OBJ2U2EA/#RJQXTDWXWGVX3XIKDJI7BYP2X2OIE632>`_
```

Comment:
I think this one is the correct link: 
https://mail.python.org/archives/list/numpy-discussion@python.org/message/PS7EN3CRT6XERNTCN56MAYOXFFFEC55G/

(confusing, because it is a reply to the other one)

Reply:
Yeah I was wondering but didn't know the full context, will fix thanks!